README.md

```diff
@@ -4,58 +4,102 @@
 [![](https://images.microbadger.com/badges/version/planckscloud/plancks-cloud.svg)](https://microbadger.com/images/planckscloud/plancks-cloud "Get your own version badge on microbadger.com")&nbsp;<a href="https://trello.com/b/NutXeZwS/plancks-roadmap"><img src="https://img.shields.io/badge/Roadmap-Trello-brightgreen.svg" /></a>
 <a href="https://coggle.it/diagram/XEgmhoO3UopF8htc/t/logo"><img src="https://img.shields.io/badge/Ideas-Coggle-brightgreen.svg" /></a>&nbsp;[![License](http://img.shields.io/:license-mit-blue.svg?style=flat)](http://badges.mit-license.org)
 
-# What is Plancks Cloud?
+# What is Planck's Cloud?
 
-Planck's Cloud is a private or public cloud that aims to turn every home into a datacenter. Host your next project from your own home with Planck's Cloud.
+Have you ever wanted to build a tiny app and host it from home? Have you ever had a startup idea and wanted to deploy it within an hour? Planck's Cloud aims to make deploying, reverse proxying and HTTPS your next idea from home a walk in the park.
```

Comment:
Planck's Cloud aims to make deploying, reverse proxying and HTTPS-ing your next idea from home a walk in the park.